cli/cli.go

```diff
@@ -331,6 +331,7 @@ documentation for detail information about writing tests.`,
 			cli.StringFlag{Name: "run-test, r", Value: "", Usage: "Run only tests matching specified regex"},
 			cli.IntFlag{Name: "parallel, p", Value: runtime.NumCPU(), Usage: "Allow parallel execution of test functions"},
 			cli.StringFlag{Name: "type, t", Value: "", Usage: "Run only specific types of tests from a comma separated list (js,go); if not specified, all types of tests are run"},
+			cli.StringFlag{Name: "schemas-file, s", Value: "gohan.yaml", Usage: "File containing schemas to load"},
```

Comment:
For the generality, can we simply use config-file to get schemas?

Reply:
true, thanks